client/k8s/k8s.go

```diff
@@ -54,8 +54,8 @@ func DefaultConfig() Config {
 		Logger: newLogger,
 
 		// Settings.
-		Address:   "http://127.0.0.1:8080",
-		InCluster: false,
+		Address:   "",
```

Comment:
Given that "" is the zero value for strings, could we remove this?

Reply:
Talk to @xh3b4sd when he's back. He likes that.